test/gtest/stringify.cpp

```diff
@@ -125,3 +123,100 @@ adding unique symbol
 
   EXPECT_EQ(ss.str(), stringifyText);
 }
+
+using HashStringifyTest = PrintTest;
+
+TEST_F(HashStringifyTest, Print) {
+  auto moduleText = R"wasm(
+    (module
+    (func $a
+      (block $block_a
+        (drop (i32.const 20))
+        (drop (i32.const 10))
+      )
+      (block $block_b
+        (drop (if (i32.const 0)
+          (i32.const 40)
+          (i32.const 5)
+        ))
+      )
+      (block $block_c
+        (drop (if (i32.const 1)
+          (i32.const 30)
+        ))
+      )
+      (block $block_d
+        (drop (i32.const 20))
+        (drop (i32.const 10))
+      )
+      (block $block_e
+        (drop (if (i32.const 1)
+          (i32.const 30)
+        ))
+      )
+      (block $block_f
+        (drop (if (i32.const 0)
+          (i32.const 30)
+        ))
+      )
+    )
+   )
+  )wasm";
+
+  Module wasm;
+  parseWast(wasm, moduleText);
+
+  std::stringstream ss;
+  HashStringifyWalker stringify = HashStringifyWalker();
+  stringify.walkModule(&wasm);
+
+  EXPECT_EQ(stringify.hashString.size(), 45u);
+
+  EXPECT_EQ(stringify.hashString,
+            (std::vector<uint64_t>{
+              0,            // function block evaluated as a whole
+              (uint64_t)-1, // separate function block from function contents
+              2,            // block_a evaluated as a whole
+              3,            // block_b evaluated as a whole
+              4,            // block_c evaluated as a whole
+              2, // block_d has the same contents as block_a, thus same symbol
+              4, // block_e has the same contents as block_c, thus same symbol
+              5, // block_f evaluated as a whole
```

Comment:
It would be nice to have all the `//` line up.

Reply:
done